packages/react-core/src/hooks/useDropZone.ts

```diff
@@ -86,17 +87,35 @@ export default function useDropZone({
     event.stopPropagation();
     setDragState('inactive');
     const files = Array.from(event.dataTransfer.files);
-    const { acceptedFiles, rejectedFiles } = filterAllowedFiles<File>(
-      files,
-      acceptedFileTypes
-    );
 
     if (isFunction(_onDrop)) {
       _onDrop(event);
     }
-    if (isFunction(onDropComplete)) {
-      onDropComplete({ acceptedFiles, rejectedFiles });
+
+    // If no DataTransfer items interface, handle as simple files
+    if (!event.dataTransfer.items) {
```

Comment:
Nit: Can we destructure dataTransfer or items/files earlier to cut down on some of this chaining verbosity?

Reply:
Good call out, updated